core/src/main/scala/kafka/server/KafkaConfig.scala

```diff
@@ -678,7 +678,7 @@ object KafkaConfig {
   "start from " + MaxReservedBrokerIdProp + " + 1."
   val MessageMaxBytesDoc = TopicConfig.MAX_MESSAGE_BYTES_DOC +
     s"This can be set per topic with the topic level <code>${TopicConfig.MAX_MESSAGE_BYTES_CONFIG}</code> config."
-  val NumNetworkThreadsDoc = "The number of threads that the server uses for receiving requests from the network and sending responses to the network"
+  val NumNetworkThreadsDoc = s"The number of threads that the server uses for receiving requests from the network and sending responses to the network. Noted: each listener (except for control plane listener) creates its own thread pool."
```

Comment:
nit: I don't think we have this term: `control plane listener` in the doc. Maybe using controller listener?

Reply:
will copy that